yb-voyager/src/tgtdb/postgres.go

```diff
@@ -331,6 +332,19 @@ func (pg *TargetPostgreSQL) GetNonEmptyTables(tables []sqlname.NameTuple) []sqln
 	return result
 }
 
+func (pg *TargetPostgreSQL) TruncateTables(tables []sqlname.NameTuple) error {
+	tableNames := lo.Map(tables, func(nt sqlname.NameTuple, _ int) string {
+		return nt.ForUserQuery()
+	})
+	commaSeparatedTableNames := strings.Join(tableNames, ", ")
+	query := fmt.Sprintf("TRUNCATE TABLE %s", commaSeparatedTableNames)
```

Comment:
Q: If we have mentioned a list with truncate, then if there are dependencies among tables, we won't need to `CASCADE` in this case?
I guess `CASCADE` is required in case of single table or if the list doesn't contain the dependent tables?

Reply:
yup, https://www.postgresql.org/docs/current/sql-truncate.html#:~:text=is%20the%20default.-,CASCADE,that%20are%20not%20listed%20in%20the%20command.%20This%20is%20the%20default.,-Notes